### PR TITLE
On startup, wait for KBFS to load

### PR DESCRIPTION
**Pull request by gabriel**
- Adds protocol method to waitForClient
- Waits for KBFS on bootstrap
- Separates KBFS notifications and registers for them after bootstrap

### CODE REVIEW

**Comment by gabriel** [comment]
@keybase/react-hackers

File: shared/actions/config/index.js [hunk 1]
```diff
@@ -91,7 +93,28 @@ function getMyFollowing (username: string): AsyncAction {
   }
 }
 
-function getExtendedStatus (): AsyncAction {
+export function waitForKBFS (): AsyncAction {
+  return dispatch => {
+    return new Promise((resolve, reject) => {
+      configWaitForClientRpc({
+        param: {clientType: 3, timeout: 10.0},
```
Comment (chromakode) [inline shared/actions/config/index.js]: :sheep: Is there a constant definition you can use here instead of the magic number `3`?

Comment (chrisnojima) [inline shared/actions/config/index.js]: https://github.com/keybase/client/blob/master/protocol/js/flow-types.js#L55

Comment (gabriel) [inline shared/actions/config/index.js]: 👍

**Pull request merged**